pyfarm/jobtypes/core/jobtype.py

```diff
@@ -890,6 +890,96 @@ def set_task_progress(self, task, progress):
                     "%r." % (task["id"], error))
                 raise
 
+    @inlineCallbacks
+    def set_task_started_now(self, task):
```

Comment:
The use case I was thinking of is if you had a known delay when starting a job on a host.  But, I see your point from the master's perspective.  Withdrawn.